plugins/jobs/src/js/components/JobStopRunModal.js

```diff
@@ -0,0 +1,86 @@
+import { Confirm } from "reactjs-components";
+import PropTypes from "prop-types";
+import React from "react";
+import ModalHeading from "#SRC/js/components/modals/ModalHeading";
+
+class JobStopRunModal extends React.Component {
+  getContentHeader(selectedItems, selectedItemsLength) {
+    let headerContent = ` ${selectedItemsLength} Job Runs`;
+    if (selectedItemsLength === 1) {
+      headerContent = "this";
+    }
+
+    return (
+      <ModalHeading key="confirmHeader">
+        {`Are you sure you want to stop ${headerContent}?`}
+      </ModalHeading>
+    );
+  }
+
+  getConfirmTextBody(selectedItems, selectedItemsLength) {
+    let bodyText;
+
+    if (selectedItemsLength === 1) {
+      bodyText = `the job run with id ${selectedItems[0]}`;
+    } else {
+      bodyText = "the selected job runs";
+    }
+
+    return <span key="confirmText">You are about to stop {bodyText}.</span>;
+  }
+
+  getModalContents() {
+    const { selectedItems } = this.props;
+    const selectedItemsLength = selectedItems.length;
+
+    return (
+      <div className="text-align-center">
+        {this.getConfirmTextBody(selectedItems, selectedItemsLength)}
```

Comment:
It appears to me that this function needs only 1 argument: `selectedItems`

Reply:
That is also true, but also some presentational logic that I am not willing to touch.